tests/unit/test_discovery_client.py

```diff
@@ -109,6 +109,7 @@ def test___get_discovery_service_address___start_service_jit___returns_expected_
         cwd=exe_file_path.parent,
         stdout=subprocess.DEVNULL,
         stderr=subprocess.DEVNULL,
+        creationflags=0x1000000,
```

Comment:
Add `sys.platform` checks and use the constant.

Reply:
I used this constant because `mypy` was throwing error for `CREATE_BREAK_AWAY_FROM_JOB`. But now we have made it as platform specific, error is no longer thrown. So I kept the attribute name itself instead of constant.